CONTRIB.rst

```diff
@@ -0,0 +1,13 @@
+Contributors
+++++++++++++
+
+This is a list of people who have made contributions to SIMBAD.
+
+.. table::
```

Comment:
Remove the table and leave it as a list (maybe with links).

https://raw.githubusercontent.com/rigdenlab/conkit/master/CONTRIB.rst